module/userProjects/helper.js

```diff
@@ -3296,7 +3298,7 @@ function _entitiesInformation(entityIds, userToken) {
 
 			if (locationIds.length > 0) {
 				let queryData = {
-					'registryDetails.locationId': { $in: locationIds },
+					_id: { $in: locationIds },
```

Comment:
@MallanagoudaB What is this change?

Reply:
i got this in conflicts

Reply:
@MallanagoudaB does this change required? check with @Prajwal17Tunerlabs

Reply:
@Prajwal17Tunerlabs added this

Reply:
@MallanagoudaB my question is which one is correct?

Reply:
'registryDetails.locationId': { $in: locationIds }, this should be present and code is not braking anywere bcause of this

Reply:
@MallanagoudaB that is not the present code right?

Reply:
no its not there

Reply:
@MallanagoudaB then we should keep the old logic

Reply:
that changes was made by @Prajwal17Tunerlabs  we need to confirm with him

Reply:
@MallanagoudaB  in the second comment its self I told you to check with him and confirm. Why you are not doing it still? connect with him and give me the correct query. Please don't make me repeat this again